Lesson Modules/Module 6- Data Binning.ipynb

```diff
@@ -98,8 +98,10 @@
     "            DataCount.append(0)\n",
     "            bin_num += 1\n",
     "    \n",
-    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)              # np.array allows us to perform element-by-elemtent division\n",
-    "    binnedError = np.array(sumError) / np.array(DataCount)\n",
+    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)          # np.array allows us to perform element-by-elemtent division\n",
+    "    \n",
+    "    avgError = np.array(sumError) / np.array(DataCount)\n",
+    "    stdError = avgError / np.sqrt(len(DataCount))               # standard error is average error divided by sqrt(# of elements)\n",
```

Comment:
Oh, I didn't catch that.  Thanks for pointing it out.  I'll update the code
and the figures.

On Aug 1, 2016 6:07 PM, "Brian Plimley" <notifications@github.com> wrote:

> In Lesson Modules/Module 6- Data Binning.ipynb
> <https://github.com/bearing/dosenet-analysis/pull/6#discussion_r73080159>:
>
> > @@ -98,8 +98,10 @@
> >      "            DataCount.append(0)\n",
> >      "            bin_num += 1\n",
> >      "    \n",
> > -    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)              # np.array allows us to perform element-by-elemtent division\n",
> > -    "    binnedError = np.array(sumError) / np.array(DataCount)\n",
> > +    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)          # np.array allows us to perform element-by-elemtent division\n",
> > +    "    \n",
> > +    "    avgError = np.array(sumError) / np.array(DataCount)\n",
> > +    "    stdError = avgError / np.sqrt(len(DataCount))               # standard error is average error divided by sqrt(# of elements)\n",
>
> Not len(DataCount), but DataCount elementwise! That contains the number
> of measurements in each month. The division is elementwise because
> avgError is already an array and np.sqrt will return an array.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bearing/dosenet-analysis/pull/6/files/62e8de312d6022bbef95992d21c3c10f9bd4c5ba#r73080159>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS_mwg5h3wprm_QMNNgJ4WosAjheb9jWks5qbphGgaJpZM4JaHyx>
> .
>